docs/README.md

```diff
@@ -1,6 +1,40 @@
 # auth-service/Docs
 
 Documentation for the auth service
+## Database
+
+The following Figure depicts the Entity Relationship Diagram of the auth-service's schema:
+
+<p align="center">
+  <img src="http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/spikerheado1234/auth-service/master/docs/ErDiagram.puml">
+</p>
+
+The following table depicts the types of each attribute in the `Consent` table along with the constraints:
+
+|Attribute| Represented Internal Type | Constraints| Description |
+|:---------:|:---------------------------:|:------------:|:---------------------------:|
+|`id`|`VARCHAR(36)`|Unique - Primary Key|Alphanumeric consent ID|
+|`initiator_id`|`VARCHAR(32)`|Non-Null|Alphanumeric PISP ID|
+|`participant_id`|`VARCHAR(32)`|Non-Null|Alphanumeric DFSP ID|
+|`credential_id`|`INT UNSIGNED`|Unique - Non-Null|Unsigned numeric ID - upto 2^32 records|
```

Comment:
I think this **needs** to be _nullable_, since it will be filled in by the PISP - it's the unique identifier of the FIDO key on the device, so won't be filled in until the PISP has signed the challenge.

Reply:
Is there a specific data type for the PISP credential ID or can the auth service specify the type of ID it wants?